sourcemash/frontend/templates/feed-card.tpl

```diff
@@ -0,0 +1,36 @@
+<div class="col m6">
+  <div class="card">
+
+    <div class="card-image waves-effect waves-block waves-light">
+      <a href="#/feeds/<%= model.get('id') %>">
+        <img class="activator mark-read" src=<%= model.get('image_url') %>>
+      </a>
+    </div>
+    
+    <div class="card-content">
+      
+      <div class="card-title">
+        <div class="col s9">
+          <a href="#/feeds/<%= model.get('id') %>" class="row">
+            <%= model.get('title') %>
+          </a>
+        </div>
+        <div class="switch col s3 valign-wrapper">
+          <label id="subscribe-switch" class="valign">
```

Comment:
Do we want to extract this out to a template since we use it in two different places?